documentation/source/troubleshooting.rst

```diff
@@ -101,3 +101,46 @@ and having them ignored is likely to lead to strange errors.
 As a side note,
 when you need to *clear* a Makefile variable from the command line,
 use the syntax ``make EXTRA_ARGS=``.
+
+``GLIBCXX_3.4.XX`` not found
+============================
```

Comment:
This heading doesn't seem to fit with the other headings in this file. Perhaps
```suggestion
Diagnosing ``ImportError``\ s of cocotb
=======================================
```
and then some subheadings for each message.

Reply:
That's too general and makes an assumption there is more than one type of that error and they are somehow related. We can group once we have more individual cases.